Bottom: b8cb9e7cbc9da8db6edfd973d3e077fe767aa9dd
Top:    b8687aaba11f225806e46e6ff6a5811118a2ffa9
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-18 23:51:06 +0100

Refresh of don-t-abuse-crosshairon-api-to

---

diff --git a/src/crosshair.c b/src/crosshair.c
index 8e1d089..7d9fcb4 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -732,9 +732,9 @@ CrosshairOff (void)
 {
   if (Crosshair.On)
     {
-      Crosshair.On = false;
       DrawAttached ();
       DrawMark ();
+      Crosshair.On = false;
     }
 }
